hantec_api_ecommerce/controllers/main.py

```diff
@@ -0,0 +1,570 @@
+
+from odoo.http import request, Response, Controller, route
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MainController(Controller):
+    @route('/create_contact', methods=['POST'], type='json', auth='user' )
+    def create_contact(self):
+        
+        # Lista de todos los campos posibles
+
+        all_fields = [
+            'company_type', 'name', 'phone', 'street_number', 'street_number2',
```

Comment:
keep as a list 
```suggestion
            'company_type', 
            'name', 
            'phone', 
            'street_number', 
            'street_number2',
```

an so on, replace single quote by double quoting